build-info-extractor/src/test/java/org/jfrog/build/extractor/packageManager/PackageManagerUtilsTest.java

```diff
@@ -103,6 +106,36 @@ public void testExcludePatterns() {
 
     }
 
+    @Test
+    public void testExcludeJfrogInternalKey() {
+        Properties props = new Properties();
+        Properties buildInfoProperties = getEnvProperties(props, new NullLog());
+        Properties moduleProps = new Properties();
+        moduleProps.setProperty(key1, value1);
+        moduleProps.setProperty(PROP_PROPS_FILE_KEY, value1);
+        moduleProps.setProperty(PROP_PROPS_FILE_KEY_IV, value1);
+        moduleProps.setProperty(PROP_PROPS_FILE, value1);
+        Module module = new Module();
+        module.setId("foo");
+        module.setProperties(moduleProps);
+        BuildInfo buildInfo = new BuildInfoBuilder("BUILD_NAME")
+                .number("BUILD_NUMBER")
+                .properties(buildInfoProperties)
+                .startedDate(new Date())
+                .properties(buildInfoProperties)
+                .addModule(module).build();
+
+        filterBuildInfoPropertiesTestHelper(buildInfo);
+
+        // Excluded build info JFrog internal keys
+        assertNull(buildInfo.getProperties().getProperty(PROP_PROPS_FILE), "Should not find '" + PROP_PROPS_FILE + "' property due to exclude JFrog internal key");
+        assertNull(buildInfo.getProperties().getProperty(PROP_PROPS_FILE_KEY_IV), "Should not find '" + PROP_PROPS_FILE_KEY_IV + "' property due to exclude JFrog internal key");
+        assertNull(buildInfo.getProperties().getProperty(PROP_PROPS_FILE_KEY), "Should not find '" + PROP_PROPS_FILE_KEY + "' property due to exclude JFrog internal key");
+
+        // Keep build info property
+        assertEquals(buildInfo.getModule("foo").getProperties().getProperty(key1), value1, key1 + " property does not match");
```

Comment:
Let's also check in the buildinfo properties, as we do for the excluded props:
```suggestion
        assertEquals(buildInfo.getModule("foo").getProperties().getProperty(key1), value1, key1 + " property does not match");
        assertTrue(buildInfo.getProperties().containsKey(key1), "Should find '" + key1 + "'");
```